workers/data_refinery_workers/downloaders/array_express.py

```diff
@@ -51,7 +51,7 @@ def _extract_files(file_path: str, accession_code: str) -> List[str]:
     """Extract zip and return a list of the raw files.
     """
 
-    logger.debug("Extracting %s!", file_path, downloader_job=job.id)
+    logger.debug("Extracting %s!", file_path, file_path=file_path)
```

Comment:
Note that this will result in a log line like:
```
<timestamp and other normal context info> [file_path: /path/to/file]: Extracting /path/to/file!
```